library/azure_rm_webappslot.py

```diff
@@ -120,6 +120,10 @@
             registry_server_password:
                 description:
                     - The container registry server password.
+    startup_file:
+        description:
+            - The slot startup file.
+            - This only applies for linux web app slot.
     app_settings:
         description:
             - Configure web app slot application settings. Suboptions are in key value pair format.
```

Comment:
so it's a dict, right?
Maybe should be "A dictionary containing Web App slot settings"?